src/waveform/renderers/waveformrendermark.cpp

```diff
@@ -107,20 +109,27 @@ void WaveformRenderMark::slotCuesUpdated() {
             continue;
         }
 
-        QString newLabel = pCue->getLabel();
-        QColor newColor = pCue->getColor();
-
         // Here we assume no two cues can have the same hotcue assigned,
         // because WaveformMarkSet stores one mark for each hotcue.
         WaveformMarkPointer pMark = m_marks.getHotCueMark(hotCue);
         if (pMark.isNull()) {
         	continue;
         }
+
         WaveformMarkProperties markProperties = pMark->getProperties();
+        QString newLabel = pCue->getLabel();
+        QColor newColor;
+        PredefinedColorPointer cueColor = pCue->getColor();
+        if (*cueColor == *Color::predefinedColorSet.noColor) {
+            newColor = markProperties.m_defaultColor;
+        } else {
+            newColor = m_predefinedColorsRepresentation.map(cueColor);
```

Comment:
It would be nice to use here just: 
cueColor.getCustomizedRgba()

Reply:
We are discussing this in another comment. I resolve this one :)